contracts/LocalOracle.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+pragma solidity 0.7.1;
+
+import "./ERC1155ERC721.sol";
+import "./interfaces/ILocalOracle.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+
+
+contract LocalOracle is ILocalOracle {
+
+    using SafeMath for uint256;
+
+    // reference for ERC1155ERC721 contract
+    ERC1155ERC721 ERC1155;
+
+    // the event which will be emitted when help is needed from other blockchains
+    event HelpMe(uint256 indexed tokenId, address indexed owner);
```

Comment:
I'm not sure Chainlink is able to extract several parameters from the event data. Need to be tested.
Maybe we would need to concatenate all the parameters in one data (kind of callData as an array of bytes).
I'll do some test this afternoon

Reply:
ideally if we can do it with separate args, that will be best, otheriwse we can look at bytes array